### ISSUE TITLE
Streamline local set up

### ISSUE THREAD

**Issue by jsstevenson** [cancervariants/metakb#282]
- [ ] Review + update documentation
- [ ] Provide Neo4j dump?
- [ ] Instructions for normalizer setup (and how to make it quicker/easier)*

*Could we just have instructions say to use the HTTP endpoints?

**Comment by katiestahl** [comment]
I agree we should streamline this. I think it would be nice to include instructions for simply pointing to deployed instances of the normalizers. (I've been getting around this with the environment variables) I think that would probably be the most common use case for development, since I don't think code changed in MetaKB would affect the normalizers (but it would the other way around, but in that case, you're likely working on one of the normalizers and not necessarily MetaKB)

**Comment by mcannon068nw** [comment]
related to #671
